src/assets/wise5/components/peerChat/peer-chat-question-bank/peer-chat-question-bank.component.ts

```diff
@@ -70,8 +70,8 @@ export class PeerChatQuestionBankComponent implements OnInit {
   ): QuestionBankRule[] {
     const cRaterResponses = peerGroupStudentData.map((peerMemberData: PeerGroupStudentData) => {
```

Comment:
Should this variable be renamed to `responses` since it can now be Multiple Choice responses too?